src/App.js

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 import logo from "./logo.svg";
 import "./App.css";
+import Items from "./test.js";
```

Comment:
So great you've started with tests from day one! They're definitely one of my bigger weaknesses at the moment. 😬

Reply:
That's just the file name we were using to verify the data will push and pull from Firebase, it can be changed if that would make more sense?